assets/src/components/routeLadder.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from "react"
+import { fetchTimepointsForRoute } from "../api"
+import { Route, Timepoint, TimepointsByRoute } from "../skate"
+import { Dispatch, setTimepointsForRoute } from "../state"
+import Loading from "./loading"
+
+interface Props {
+  route: Route
+  dispatch: Dispatch
+  timepointsByRoute: TimepointsByRoute
```

Comment:
```suggestion
  timepoints: Timepoint[] | null
```
If we do the lookup before we call this component, then it doesn't need to know anything about other ladders.

Reply:
👍 I actually started to do this at one point but it got lost in the shuffle. Added a function.